packages/phishing-controller/CHANGELOG.md

```diff
@@ -28,7 +31,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
     All changes listed after this point were applied to this package following the monorepo conversion.
 
-[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/phishing-controller@1.1.1...HEAD
+
```

Comment:
Hmm, I would expect the linter to complain about this newline as well, but I'm not entirely sure

```suggestion
```